pandas/core/series.py

```diff
@@ -1462,63 +1463,21 @@ def dot(self, other):
         else:  # pragma: no cover
             raise TypeError('unsupported type: %s' % type(other))
 
-    def searchsorted(self, v, side='left', sorter=None):
-        """Find indices where elements should be inserted to maintain order.
+    @Substitution(klass='Series')
+    @Appender(base._shared_docs['searchsorted'])
+    def searchsorted(self, v, side='left', *args):
+        validate_args(args, min_length=0, max_length=1)
+        sorter = None if not args else args[0]
 
-        Find the indices into a sorted Series `self` such that, if the
-        corresponding elements in `v` were inserted before the indices, the
-        order of `self` would be preserved.
+        if not isinstance(self._values, Index):
+            if sorter is not None:
+                sorter = com._ensure_platform_int(sorter)
 
```

Comment:
sorry for the back and forth, but, @jreback you may like it or not, but we *do* support `sorter` for `pd.Series.searchsorted`: it is documented in the docstring and it is explicitly passed through to numpy's searchsorted

So we either remove it, but then it's an API change and should wait until 0.19, or we just keep it. And to be honest, I don't see the problem in keeping it. It is just simply passing it to numpy's method (it does not make the code any more complicated), and we have to accept it anyway regardless of if we pass it through or not (for the compat issues that started this PR)